### DISCUSSION TITLE
String concatenation in ternary operator

### DISCUSSION THREAD

**Discussion by jarnalyrkar** [statamic/cms#4751] [Troubleshooting] [answered]
I would like to write something like this:
`{{ custom_title ? custom_title : "Featured: " ~ other_title }}`

I've tried php, js and twig syntax, but no cigar.
Do I have to write an if-statement?

**Comment by jackmcdade** [comment, marked as answer]
This is the best I could get for ya here:

```
{{ custom_title ? custom_title : other_title | ensure_left:Featured&colon;&nbsp; }}
```

BUT, you should be able to do it more-or-less the way you're doing it now as soon as this PR is merged: <https://github.com/statamic/cms/pull/4257>

**Reply by jarnalyrkar** [reply]
Thank you!
